### DISCUSSION TITLE
2023 年终总结 | Mengke's blog - Mengke's coding journey

### DISCUSSION THREAD

**Discussion by giscus** [mk965/mengke.me#5] [Announcements]
# 2023 年终总结 | Mengke's blog - Mengke's coding journey

稍纵即逝的一年即将结束。今年我也顺应潮流，记录年终总结。过去，我对「年终总结」不屑一顾，认为人生就是不断前行，有欢笑有泪水，一切都将过去。年纪稍长，渐渐明白，一些事情漫不经心就可能被遗忘，需要人为地留下痕迹...

https://www.mengke.me/blog/202401/2023_Year_Summary

**Comment by wangzi0218** [comment]
好帅！

**Reply by mk965** [reply]
你是有眼光的哈哈哈👍

**Comment by successjia** [comment]
咱俩是校友，哈哈哈qusters

**Reply by mk965** [reply]
quster 你好 🤝

**Comment by successjia** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
